tools/cr/repository.py

```diff
@@ -144,6 +144,9 @@ def read_file(self, *files, commit: str = 'HEAD') -> str:
         The contents of the file read. If more than one file is provided, the
         contents of all files are appended to the same string.
         """
+
+        # fails here! so before running git show, replaces \ path with /.
+        files = [str(file).replace('\\', '/') for file in files]
```

Comment:
I think the correct way to handle this in python is to use `as_posix` down below.

https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix